### DISCUSSION TITLE
Good practice: Nuxt App + Nitro Server + Rest API

### DISCUSSION THREAD

**Discussion by fpsilva98** [nuxt/nuxt#28287] [General]
Considering that I already have a java spring boot application.
Is it a good practice to have a Nuxt application that makes calls to the Nitro server api and it in turn makes calls to the java API? Is safer?
I have read in some places that direct calling is not recommended.

**Comment by masyoudi** [comment]
for several security reasons, i think is good to implement proxy API, because attacker doesn't to know which one is the core REST API
